protos/protos/public/modeldb/metadata/MetadataService.proto

```diff
@@ -34,6 +34,14 @@ message GetLabelsRequest {
     }
 }
 
+message GetLabelIdsRequest {
+    repeated string labels = 1;
```

Comment:
is this better off a key-value query? or at least have operator included for negative filters?

Reply:
Resolved conversion via slack talk.